### PR TITLE
NEPT-1680: Made newsletters translatable.

### PR DESCRIPTION
**Pull request by gillesdeudon**
### Description

Made newsletters translatable.

### Change log

- Changed: Made newsletters translatable.

### CODE REVIEW

**Review by gillesdeudon — COMMENTED** [review]
I did the PR but also the code review on the previous broken PR. It is ok for me.

File: profiles/common/modules/features/newsletters/newsletters.install [hunk 1]
```diff
@@ -19,10 +21,13 @@ function newsletters_enable() {
 }
 
 /**
- * Use soft config for newsletters comment form location.
+ * Use soft config for comment form location and simplenews content type.
  */
 function newsletters_install() {
-  multisite_config_service('comment')->setThreadingCommentForContentType('simplenews', 1);
+  // Activate entity translation for simplenews content type.
+  variable_set('language_content_type_simplenews', ENTITY_TRANSLATION_ENABLED);
```
Comment (Fefaine) [inline profiles/common/modules/features/newsletters/newsletters.install]: I don't like the idea of forcing ET at the install of the feature. What if someone wants to use it and does not want to use entity translation ? 42 sites (at least) have disabled ET

**Review by Fefaine — CHANGES_REQUESTED** [review]

**Review by aritomelo — APPROVED** [review]
Looks ok for me. Next to +1

**Review by ventzie555 — APPROVED** [review]
Code looks good. I also tested it  and it looks like it the issue is fixed.

**Review by Fefaine — APPROVED** [review]

**Pull request merged**